hw05.Rmd

```diff
@@ -0,0 +1,114 @@
+---
+title: "hw05"
+author: "Maria Evers"
+date: "8/3/2020"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+## R Markdown
+
+This is an R Markdown document. Markdown is a simple formatting syntax for authoring HTML, PDF, and MS Word documents. For more details on using R Markdown see <http://rmarkdown.rstudio.com>.
+
+
+##Homework 5 part 1:
+```{r}
+
+library(tidyverse)
+library(readr)
+library(knitr)
+library(stringr)
+library(dplyr)
+
+#loading in the data 
+
+dad_mom <- read_delim("dad_mom.txt", delim = "\t")
+
+#tidying the data 
+
+dad_mom_tidy1 <- dad_mom %>%
+  
+#tidying columns 
+  
+  gather(key = "parent", value = "name_parent", name_dad, name_mom) %>%
+  
+#renaming mom and dad 
+  
+  mutate(parent = recode(parent, 'name_dad' = "dad", 'name_mom' = "mom")) %>%
+  
+#pairing the income with the correct name 
+  
+  mutate(income = ifelse(parent == "dad", income_dad, income_mom)) %>%
+
+#since we previously used the mutate(ifelse) function to assort the income to each person, we can get rid of income_mom and income_dad for each family ID because this is redundant.   
+  
+  select(-income_mom, -income_dad) 
+
+    
+
+```
+
+##Homework 5 part 2:
+```{r}
+
+
+library(ggplot2)
+
+
+#Reading in CTRP files from CTRP Folder as CSV files 
+
+AUC_1 <- read.csv("CTRP_files/AUC_1.csv")
+AUC_2 <- read.csv("CTRP_files/AUC_2.csv")
+cancer_cell_line_info <- read.csv("CTRP_files/cancer_cell_line_info.csv")
+compound_info <- read.csv("CTRP_files/compound_info.csv")
+experiment_info <- read.csv("CTRP_files/experiment_info.csv")
+
+#Combine data- start by binding AUC1 and AUC2 since they have the exact same columns. Then, combine the rest by their common columns one by one using inner_join
```

Comment:
I should have just use bind_rows() like you did. I just joined everything together but ended up with extra columns that only had NA which I had to remove.